internal/testbed/integration/k8senrichment/README.md

```diff
@@ -15,3 +15,77 @@ on the k8s cluster with configurations as per the Dynatrace documentation page.
 Traces are generated and sent to the Collector, which then
 exports to the test where the k8s attributes are asserted on the
 received traces.
+
+## Running the tests locally
+
+To execute the tests locally, follow these steps:
+
+### Start a `kind` cluster
+
+To create a new cluster, execute the following command:
+
+```shell
+kind create cluster
+```
+### Build the collector binary
+
+This is done using the `make build` command.
+This will build the collector distro, and place the built binary 
+into the `bin` directory of your local copy of the repository.
+
+**NOTE:** When using an M1 mac, the `make build` command will build the `arm64` binary, which
+will not be able to run as a container on a `kind` cluster.
+For now, as workaround you will need to use the `make build-all` command to build all binaries and then copy the
+`linux_amd64` binary from the `dist` directory into `bin` under the name `dynatrace-otel-collector`.
```

Comment:
why is the ARM version not working? shouldn't that be used for ARM based Macs?

Reply:
I do not know - I just found that for now this is what I had to do to make this run in my setup. Investigating this further could be done in a follow up to make the local setup a bit easier though